src/sonolus/configuration.ts

```diff
@@ -0,0 +1,15 @@
+import { ServerOptionsModel } from '@sonolus/express'
+
+export const serverConfiguration = {
+    spoilers: {
+        type: 'toggle',
+        name: {
+            en: 'Spoilers',
+        },
+        def: false,
```

Comment:
Should show spoilers by default, as most people do not care about getting spoiled.

Reply:
I think unreleased items should be hidden by default, as this is intended behavior.
Also failing to find something wanted (and should not be seen), is better than seeing something unwanted.

Reply:
It seems like Sekai Viewer also has spoiler off by default.